backend/Generator/question_generators.py

```
@@ -0,0 +1,239 @@
+        self.device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+        self.model.to(self.device)
+        self.set_seed(42)
+        
+    def set_seed(self, seed):
+        np.random.seed(seed)
+        torch.manual_seed(seed)
+        if torch.cuda.is_available():
+            torch.cuda.manual_seed_all(seed)
+
+    def random_choice(self):
+        a = random.choice([0,1])
```

Ruff (0.15.6)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)